src/components/ArticleDetail.jsx

```diff
@@ -16,19 +17,15 @@ function ArticleDetail() {
         setArticle(article)
       }
 
-    function handleClick(event) {
-        event.preventDefault()
-        navigate(`/articles/${article.article_id}/comments`)
-    }
-
     // useEffect callback to invoke the get method. 
     useEffect( () => {getArticleSetState(article_id)}, [])
 
     return (
         <div className='article_detail'>
             <ArticleSummary key={article.article_id} article={article} />
-            <p>{article.body}</p>
-            <button onClick={handleClick}>View Comments</button>
+            <h2>{article.body}</h2>
+            {/* <button onClick={handleClick}>View Comments</button> */}
```

Comment:
This might be a nice feature to add back in at some point so the user can toggle the article comments between hidden and shown. Consider how you might flip the text between "view comments" and "hide comments"